src/me/StevenLawson/TotalFreedomMod/TFM_Util.java

```diff
@@ -1003,6 +1009,27 @@ public static void reportAction(Player reporter, Player reported, String report)
             }
         }
     }
+    
+    //Note that in order for the titles to work properly, the sub-title must follow the title packet.
+    public static void sendTitle(Player player, String message, int fadein, int stay, int fadeout)
+    {
+        CraftPlayer craftplayer = (CraftPlayer) player;
```

Comment:
In case this fails casting (running on some other platform), fail gracefully. :)